### DISCUSSION TITLE
Anyone know how to add Latex support?

### DISCUSSION THREAD

**Discussion by hellochang** [daattali/beautiful-jekyll#820] [General]
I tried adding the line ```<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>``` in ```post.html```, but it didn't work. Is there a specific place that I need to add MathJax for the Beautiful Jekyll theme?

**Comment by daattali** [comment]
There's a discussion here, hopefully it's helpful https://github.com/daattali/beautiful-jekyll/issues/195

**Reply by hellochang** [reply]
I think it was working for rendering latex, but I was more referring to rendering $$latex formula$$ for LaTeX (not sure why but the proposed solution wasn't working for me, like what was happening with the last comment in the thread).
